### PR TITLE
[IMP] iot: admonition about the required certification to use a scale in POS

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
task-3876833

### CODE REVIEW

**Comment by qle-odoo** [comment]
it is an european directive
EU Directive 2014/31/EU

Check this https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv%3AOJ.L_.2014.096.01.0107.01.ENG

File: content/applications/general/iot/devices/scale.rst [hunk 1]
```diff
@@ -6,6 +6,13 @@ A scale can be connected to the :abbr:`IoT (Internet of Things)` box on an Odoo
 easy steps. After setup, the *Point of Sale* app can be used to weigh products, which is helpful if
 their prices are calculated based on weight.
 
+.. important::
+   - In Europe, `certification is legally required <https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv%3AOJ.L_.2014.096.01.0107.01.ENG>`_
```
Comment (xpl-odoo) [inline content/applications/general/iot/devices/scale.rst:10]: I would specify it applies to EU member states. Europe is too vague :)

File: content/applications/general/iot/devices/scale.rst [hunk 2]
```diff
@@ -6,6 +6,13 @@ A scale can be connected to the :abbr:`IoT (Internet of Things)` box on an Odoo
...
+     for using the scale as an integrated device. However, users can still utilize it without
+     integration.
+   - Alternatively, they have the option to acquire a certified scale with a printed certification
```
Comment (xpl-odoo) [inline content/applications/general/iot/devices/scale.rst:13]: Unless I'm missing something, it feels like you are talking to someone implementing Odoo for its customers ("they"/"users"), and I believe we should address the documentation to the end-users instead.

**Comment by jcs-odoo** [comment]
@qle-odoo @LoredanaLrpz @xpl-odoo 
That was my point (which is not written here ^^'): the way it's written "In Europe, certification is legally required", I'm not sure to understand if Odoo isn't certified, if the scale needs a certification, if the business needs a certification... I thought that the point was that French users (and some other countries) couldn't use _any_ scale directly connected to an Odoo database... Unless I misunderstood the point here.

**Review by jcs-odoo — APPROVED** [review]
@LoredanaLrpz @odoo/iot-doc-review 

Merging this now because we need this online asap, but it will need to be clarified and more detailed. Please keep an eye on this.

Cheers and have a good day!

@robodoo r+

**Comment by jcs-odoo** [comment]
@robodoo r-

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**